snapshots/testsuite/testsuite.go

```diff
@@ -269,10 +268,16 @@ func checkSnapshotterBasic(ctx context.Context, t *testing.T, snapshotter snapsh
 	}
 
 	testutil.Unmount(t, nextnext)
-	assert.NilError(t, snapshotter.Remove(ctx, nextnext))
-	assert.Assert(t, is.ErrorContains(snapshotter.Remove(ctx, committed), "remove"))
-	assert.NilError(t, snapshotter.Remove(ctx, nextCommitted))
-	assert.NilError(t, snapshotter.Remove(ctx, committed))
+	assert.Nil(t, snapshotter.Remove(ctx, nextnext))
+
+	err = snapshotter.Remove(ctx, committed)
+	assert.NotNil(t, err)
+	if err != nil {
+		assert.Contains(t, err.Error(), "remove")
+	}
```

Comment:
I think this can be simplified

```suggestion
	assert.ErrorContains(t, err, "remove")
```